docs/src/main/asciidoc/hibernate-orm-panache-guide.adoc

```diff
@@ -440,3 +452,34 @@ your entity `Person` by typing `Person.` and getting completion for all the oper
 `Person.find("name", "stef")`.
 
 That's all there is to it: with Panache, Hibernate ORM has never looked so trim and neat. 
+
+== Defining entities in external projects or jars
+
+Hibernate ORM with Panache in Quarkus relies on compile-time bytecode enhancements to your entities. If you define your entities in the
+same project where you build your Quarkus application, everything will work fine. If the entities come from external projects
+or jars, you should build those jars with the Jandex Maven plugin, so that your entities will be indexed and found by Quarkus
+and enhanced:
+
+[source,xml]
+--
+<plugin>
+    <groupId>org.jboss.jandex</groupId>
+    <artifactId>jandex-maven-plugin</artifactId>
+    <version>1.0.3</version>
+    <executions>
+        <execution>
+            <id>make-index</id>
+            <goals>
+                <goal>jandex</goal>
+            </goals>
+        </execution>
+    </executions>
+    <dependencies>
+        <dependency>
+            <groupId>org.jboss</groupId>
+            <artifactId>jandex</artifactId>
+            <version>2.1.1.Final</version>
+        </dependency>
+    </dependencies>
```

Comment:
Then this dependency can go away as `1.0.6` includes the right dependency.

Reply:
Thanks, fixed